src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -642,16 +670,18 @@ def _format_batch_size_and_grad_accum_config(self):
         if "gradient_clipping" not in self.config:
             self.config["gradient_clipping"] = self.lightning_module.trainer.gradient_clip_val or 0.0
 
-    def _auto_select_batch_size(self):
+    def _auto_select_batch_size(self) -> Optional[int]:
```

Comment:
when is it optional?

Reply:
Yeah, this should not be optional. Fixed and thanks!